### ISSUE TITLE
Confusing variants count in OC

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#5001]
## Description

We have recently improved our count of variants, but there still are some weird cases.

Here is an example with french prod shop 173  :

1. The list of OC shows 65 variants:

![image](https://user-images.githubusercontent.com/6525576/76757761-86066280-6788-11ea-84e6-4799277693de.png)

2. The sum in incoming is correct.

3. In outgoing it says that I have selected 64 variants. When I unfold the list it says that it is showing me 65 out of 64 variants...

![image](https://user-images.githubusercontent.com/6525576/76757623-2dcf6080-6788-11ea-9bbf-96cf6480bf2a.png)

## Steps to Reproduce
1. Go to `/admin/order_cycles` on French production
2. Have a look at OC 3063 for shop 173 

## Severity
bug-s3: a feature is broken but there is a workaround

## Your Environment
* Version used: v2.7.8
* Browser name and version: Firefox 73
* Operating System and version (desktop or mobile): Desktop

**Comment by RachL** [comment]
Ok, because I'm a big cheater, I will use this one as an epic to group the issues on this currently in dev ready.

**Comment by luisramos0** [comment]
The problem described in this issue is not seen here any longer:
https://openfoodfrance.org/admin/order_cycles/3063/outgoing

Anyway, this epic's description looks like a duplicate of #4806 (one of the issues inside it).

**Comment by luisramos0** [comment]
@RachL fyi I have just fixed this problem
![image](https://user-images.githubusercontent.com/1640378/82910005-f4725880-9f61-11ea-93d1-4506682b2e7e.png)

here in transifex (missing space):
https://www.transifex.com/open-food-foundation/open-food-network/translate/#fr/enyml/194011289?q=key%3Avariants_loaded

**Comment by RachL** [comment]
> Anyway, this epic's description looks like a duplicate of #4806 (one of the issues inside it).

Yes I cheated a bit when creating the epic and forgot to update the description 😁

**Comment by luisramos0** [comment]
Adding feedback needed to make it clear this is blocked by the pending decision in #4806

**Comment by audez** [comment]
Hi! We have a hub manager that raised the problem (the fact that in step 3 the numbers didn't make sense, ex: 4 of 0 Variants Loaded)
